cime_config/usermods_dirs/NEON/defaults/shell_commands

```diff
@@ -23,5 +24,25 @@ else
   ./xmlchange CLM_NML_USE_CASE="2018_control"
 fi
 
+# If needed for SP simulations: turn off lightning & set history file variables
+if  [[ $compset == *'SP_SICE'* ]]; then
+  ./xmlchange STOP_OPTION=nyears
+  ./xmlchange CLM_FORCE_COLDSTART=on
+  # comment out lightning file
+  :> user_nl_clm.tmp
+  while read line; do
+    if [[ $line != *"_lightng"* ]]; then
+        echo $line>>user_nl_clm.tmp
+    else
+  echo '!'$line>>user_nl_clm.tmp
+    fi
+  done<user_nl_clm
+  mv user_nl_clm.tmp user_nl_clm
+
+  echo "hist_mfilt(2) = 48" >> user_nl_clm
```

Comment:
@wwieder these two strictly aren't required because it's already set this way. Did you want this set here in case it's desired to change it for SP cases? I just wanted to understand the reasoning.

The difference in hist_fincl2 is required, as the list of history fields does need to be different between SP and BGC (and FATES for that matter).

Reply:
Good point, I've removed this